modules/virt-hot-plugging-memory.adoc

```diff
@@ -20,5 +20,6 @@ The system applies these changes immediately. If the VM is migratable, a live mi
 
 [NOTE]
 ====
-Linux guests require a kernel version of 5.16 or later and Windows guests require the latest `viomem` drivers.
+Memory hotplug for virtual machines requires guest operating systems to support the `virtio-mem` driver. Instead of relying on specific upstream kernel versions, support depends on whether the driver is included and enabled in the guest OS.
+For {op-system-base-full} guests, refer to the official support matrix to determine which RHEL versions and architectures include support for memory hotplug. 

```

Comment:
Please create a link to the matrix, as Catherine advised in her comment: https://access.redhat.com/articles/certified-hypervisors?extIdCarryOver=true&sc_cid=701f2000001Css5AAC#ocpvirt